src/main/java/org/jabref/gui/preferences/ai/AiTab.java

```diff
@@ -28,6 +29,7 @@
 
 public class AiTab extends AbstractPreferenceTabView<AiTabViewModel> implements PreferencesTab {
     private static final String HUGGING_FACE_CHAT_MODEL_PROMPT = "TinyLlama/TinyLlama_v1.1 (or any other model name)";
+    private static final String GPT_4_ALL_CHAT_MODEL_PROMPT = "Phi-3.1-mini (or any other local model name from Gpt4All)";
```

Comment:
```suggestion
    private static final String GPT_4_ALL_CHAT_MODEL_PROMPT = "Phi-3.1-mini (or any other local model name from GPT4All)";
```

Consistency! Use `GPT4All`.

Yes, it's tedious, but it gives good results